tests/unit/windows/CMakeLists.txt

```
@@ -15,7 +15,7 @@ target_link_libraries(nearobject-test-windows
         nearobject-service-windows
         notstd-windows
         uwb
-        windevuwb
+        libuwbcxadapter
```

The dependency on `windevuwb` should be restored here. The reason is that we're testing stuff in that library directly in this executable. Even though `libuwbcxadapter` depends on `windevuwb`, not specifying it adds risks to rely on indirect dependencies in case we ever change things (eg. if we decide to split the tests into separate binaries, we'll hit linker issues since not all the explicit depends are listed).

Also, the `lib` prefix isn't needed; the linker automatically prepends dependency names with `lib` when attempting to link, and it is common practice not to specify the prefix.

```suggestion
        uwbcxadapter
        windevuwb
```